bitnami/mongodb-sharded/templates/config-server/config-server-statefulset.yaml

```
@@ -54,6 +54,9 @@ spec:
       {{- if .Values.configsvr.tolerations }}
       tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.configsvr.tolerations "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.configsvr.priorityClassName }}
+      priorityClassName: {{ .Values.configsvr.priorityClassName }}
```

```suggestion
      priorityClassName: {{ .Values.configsvr.priorityClassName | quote }}
```

Can you `quote` this and the other `priorityClassName`s?

---

No problem - code updated.